projects/packages/forms/src/blocks/contact-form/components/jetpack-field-consent.js

```diff
@@ -30,8 +30,9 @@ const JetpackFieldConsent = ( {
 			<JetpackFieldLabel
 				required={ false }
 				label={
+					// @TODO: remove prop as unused?
```

Comment:
This label was a bit more complicated, so needs understanding more what's going on before removing